### PR TITLE
Add links to dotnet runtime repo in docs

### PR DESCRIPTION
**Pull request by cijothomas**
We could use some improvements to make the messaging more consistent across all signals as a future improvement. This PR is just adding links to the runtime repo.

### CODE REVIEW

File: docs/metrics/getting-started/README.md [hunk 1]
```diff
@@ -87,9 +87,9 @@ the configuration for metrics like `Meter` names, readers, etc. and is highly
 ## OpenTelemetry .NET special note
 
 Metrics in OpenTelemetry .NET is a somewhat unique implementation of the
-OpenTelemetry project, as most of the Metrics API are incorporated directly
-into the .NET runtime itself. From a high level, what this means is that you
-can instrument your application by simply depending on
+OpenTelemetry project, as most of the Metrics API are made part of the [.NET
+runtime](https://github.com/dotnet/runtime) itself. From a high level, what this
```
Comment (cijothomas) [inline docs/metrics/getting-started/README.md:91]:
```suggestion
OpenTelemetry project, as most of the Metrics API are offered from the [.NET
runtime](https://github.com/dotnet/runtime) itself. From a high level, what this
```

?

Comment (reyang) [reply docs/metrics/getting-started/README.md:91]: Maybe "built into"? I'm not good at English, maybe getting some suggestion from native English speakers?

**Review by reyang — APPROVED** [review]

File: docs/metrics/getting-started/README.md (hunk 1 above)
Comment (alanwest) [reply docs/metrics/getting-started/README.md:91]:
```suggestion
OpenTelemetry project, as most of the
[Metrics API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md)
is implemented by the [.NET
runtime](https://github.com/dotnet/runtime) itself. From a high level, what this
```

**Review by alanwest — APPROVED** [review]

**Comment by alanwest** [comment]
3000!

File: docs/metrics/getting-started/README.md [hunk 2]
```diff
@@ -87,9 +87,11 @@ the configuration for metrics like `Meter` names, readers, etc. and is highly
 ## OpenTelemetry .NET special note
 
 Metrics in OpenTelemetry .NET is a somewhat unique implementation of the
-OpenTelemetry project, as most of the Metrics API are incorporated directly
```
Comment (ThomsonTan) [inline docs/metrics/getting-started/README.md:90]: Change ".NET Core SDK" to ".NET SDK" in line 3?

Comment (cijothomas) [reply docs/metrics/getting-started/README.md:90]: Thanks. a good change for a next PR.

**Pull request merged**